app/models/agents/user_location_agent.rb

```diff
@@ -42,6 +44,7 @@ def default_options
         'secret' => SecureRandom.hex(7),
         'max_accuracy' => '',
         'min_distance' => '',
+        'api_key' => '',
```

Comment:
The api key is not required for the core functionality of the agent to work but for the map to display the location. My reasoning was to make the api key not required so someone can use the agent (possibly test it) without having to register with Google to get a key.